tests/test_element.py

```diff
@@ -51,6 +51,18 @@ def test_props_parsing():
     assert ui.element._parse_props('hint="Your \\"given\\" name"') == {'hint': 'Your "given" name'}
     assert ui.element._parse_props('input-style="{ color: #ff0000 }"') == {'input-style': '{ color: #ff0000 }'}
 
+    assert ui.element._parse_props('alsotrue=""') == {'alsotrue': True}
+    assert ui.element._parse_props("alsotrue=''") == {'alsotrue': True}
+
+    assert ui.element._parse_props("hint='Your \\\"given\\\" name'") == {'hint': 'Your "given" name'}
+    assert ui.element._parse_props("one two=1 three='abc def'") == {'one': True, 'two': '1', 'three': 'abc def'}
+    assert ui.element._parse_props('''three='abc def' four="hhh jjj"''') == {'three': 'abc def', 'four':'hhh jjj',}
+    assert ui.element._parse_props('''foo="quote'quote"''')=={"foo":"quote'quote"}
+    assert ui.element._parse_props('''foo='quote"quote' ''')=={"foo":"quote'quote"}
```

Comment:
The problem is that JSON only accepts double quotes. Since the JSON-Module was imported from nicegui, I kind of assumed that this in mandatory. I now changed to a secure eval from ast:
https://stackoverflow.com/questions/4162642/single-vs-double-quotes-in-json 
Is this better?